tools/wakebox/wakebox.cpp

```diff
@@ -72,6 +72,14 @@ void print_help() {
       "  \n"
       "    -i --allow-interactive   Use default stdin, ignoring the params json file's stdin "
       "value.  \n"
+      "    -b --bind DIR1:DIR2      Place the directory (or file) at DIR1 within the command's "
+      "view  \n"
+      "                             of the filesystem at location DIR2.                            "
+      "  \n"
+      "                             May be specified multiple times.                               "
+      "  \n"
+      "                             These are are applied after the params file mount ops.         "
```

Comment:
```suggestion
      "                             These are applied after the params file mount ops.         "
```

Reply:
I ended up rebasing this PR on top of https://github.com/sifive/wake/pull/1150, where I fixed up the line wrapping first, but I just wanted to mention that I did fix this typo as well.